ddp-workspace/projects/ddp-sdk/src/lib/services/serviceAgents/invitationServiceAgent.service.ts

```diff
@@ -0,0 +1,48 @@
+import { Inject, Injectable } from '@angular/core';
+import { HttpClient, HttpErrorResponse } from '@angular/common/http';
+import { NotAuthenticatedServiceAgent } from './notAuthenticatedServiceAgent.service';
+import { ConfigurationService } from '../configuration.service';
+import { LoggingService } from '../logging.service';
+import { EMPTY, Observable, throwError } from 'rxjs';
+import { catchError, map, mergeMap } from 'rxjs/operators';
+import { InvitationCheckPayload } from '../../../../../ddp-testboston/src/app/models/invitationCheckPayload';
+import { DdpError } from '../../../../../ddp-testboston/src/app/models/ddpError';
+import { ErrorType } from '../../../../../ddp-testboston/src/app/models/errorType';
```

Comment:
It looks like `InvitationCheckPayload`, `DdpError`, `ErrorType` should be part of the SDK, we shouldn't import anything from the app to the library.